src/cross_module_fn.c

```diff
@@ -555,6 +555,7 @@ TSDLLEXPORT CrossModuleFunctions ts_cm_functions_default = {
 	.hypertable_distributed_set_replication_factor = error_no_default_fn_pg_community,
 	.update_compressed_chunk_relstats = update_compressed_chunk_relstats_default,
 	.health_check = error_no_default_fn_pg_community,
+	.mn_get_foreign_join_paths = NULL,
```

Comment:
Following the pattern here, you might want to add a dummy function instead (I guess one that does nothing by default). Then you can avoid a NULL check when using the function.

Reply:
@erimatnor Good point. I introduced a dummy function.

Reply:
@erimatnor Good point. I introduced a dummy function.